js/modules/k6/browser/common/execution_context.go

```diff
@@ -113,6 +113,14 @@ func (e *ExecutionContext) adoptBackendNodeID(backendNodeID cdp.BackendNodeID) (
 		return nil, fmt.Errorf("resolving DOM node: %w", err)
 	}
 
+	// This can occur due to race conditions between trying to click on an element
+	// and chrome moving on (e.g. navigating).
+	if remoteObj == nil {
+		return nil, fmt.Errorf(`the page may have navigated away or the element is
+			now missing. It might happen when k6 and/or Chrome are overloaded. You
+			might need to increase the compute resources: %w`, err)
```

Comment:
Isn't `err` always nil in that case?

Reply:
Good spot, removed in 7d4fa08e59274c40af949037688a351ee676e346